docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -19,75 +19,75 @@ helpviewer_keywords:
 
-* The **symbol group** that the rule applies to, for example, public members or private fields.
-* The **naming style** to associate with the rule, for example, that the name must be capitalized or start with an underscore.
-* The severity for enforcing the convention.
+1. The **symbol group** that the rule applies to; for example, public members or private fields.
+1. The **naming style** to apply with the rule; for example, the name must be capitalized or start with an underscore.
+1. The **naming rule**, linking
+   - *symbol group*
+   - *naming style*
+   - *severity* (enforcing the convention)
 
-First, you must specify the symbol group and naming style and give each of them a title. Then you specify the naming rule, which links everything together.
+   for enabling the rule.
 
 ## General syntax
 
-To define a naming rule, symbol group, or naming style, set one or more properties using the following syntax:
+To define a *symbol group*, *naming style*, or *naming rule* component, use the following general syntax:
 
 ```ini
 <kind>.<title>.<propertyName> = <propertyValue>
 ```
 
 Each property should only be set once, but some settings allow multiple, comma-separated values.
 
-The order of the properties is not important.
-
 ### \<kind>
 
-**\<kind>** specifies which kind of element is being defined&mdash;naming rule, symbol group, or naming style&mdash;and must be one of the following:
+**\<kind>** specifies which component is being defined &ndash;&nbsp;[symbol group](#symbol-group-properties), [naming style](#naming-style-properties), or [naming rule](#naming-rule-properties)&nbsp;&ndash; and must be one of the following:
 
-| To set a property for | Use the \<kind> value | Example |
+| Component | \<kind> value | Example |
 | --- | --- | -- |
+| Symbol group | `dotnet_naming_symbols` | `dotnet_naming_symbols.interfaces.applicable_kinds = interface` |
+| Naming style | `dotnet_naming_style` | `dotnet_naming_style.pascal_casing.capitalization = pascal_case` |
 | Naming rule | `dotnet_naming_rule` | `dotnet_naming_rule.types_should_be_pascal_case.severity = suggestion` |
-| Symbol group | `dotnet_naming_symbols` | `dotnet_naming_symbols.interface.applicable_kinds = interface` |
-| Naming style | `dotnet_naming_style` | `dotnet_naming_style.pascal_case.capitalization = pascal_case` |
-
-Each type of definition&mdash;[naming rule](#naming-rule-properties), [symbol group](#symbol-group-properties), or [naming style](#naming-style-properties)&mdash;has its own supported properties, as described in the following sections.
 
 ### \<title>
 
-**\<title>** is a descriptive name you choose that associates multiple property settings into a single definition. For example, the following properties produce two symbol group definitions, `interface` and `types`, each of which has two properties set on it.
+**\<title>** is an arbitrary descriptive name you choose for assigning multiple property settings to a single component definition.
```

Comment:
I think this sounds as though there's a component definition somewhere else in the file which includes the title, and the title within the property setting refers to the component definition title. We should be getting away from this, and emphasizing that there is no independent element definition; the only element definition is the property settings themselves.